hack/e2e-internal/e2e-up.sh

```diff
@@ -29,16 +29,40 @@ source "${KUBE_ROOT}/cluster/kube-util.sh"
 
 prepare-e2e
 
+function calc_next_cidr {
```

Comment:
@madhusudancs could we do something more readable/ less convoluted here, given our simple use case? 

```sh
MAX_NET_NUMBER=255
CURRENT_NET_NUMBER=1

function calc_next_cidr {
     CURRENT_CIDR="10.180.${CURRENT_NET_NUMBER}.0/16"
     CURRENT_NET_NUMBER=$(( CURRENT_NET_NUMBER+1 ))
}

```
Also, why use a /14 address space? Generally in these cases /16 is as large as you'd even need to go, and I'd even say we could just assume that for simplicity's sake.

A person attempting to decipher this must be **very** comfortable with bash and cidr notation to even begin to understand what you have now. Mad props for the octal foo, but I think it's way overkill and going to confuse way more people than it helps in this situation.